packages/sync/src/modules/Posts.php

```diff
@@ -642,4 +642,39 @@ public function expand_post_ids( $args ) {
 			$previous_interval_end,
 		);
 	}
+
+	/**
+	 * Gets a list of minimum and maximum object ids for each batch based on the given batch size.
+	 *
+	 * @param int $batch_size The batch size for objects.
+	 *
+	 * @return array An array of min and max ids for each batch.
+	 */
+	public function get_min_max_object_ids_for_batches( $batch_size ) {
```

Comment:
Feels unfortunate to have to define the entire method again just for a different `$where_sql`. 

To prevent this, I'd recommend:

* Adding an optional `$where_sql` argument to `Module::get_min_max_object_ids_for_batches`. Empty by defaullt.
* Keeping this method defining the `$where_sql`.
* Changing this method to use `parent::get_min_max_object_ids_for_batches()` with the `$where_sql`.

Let me know if that makes sense.

Reply:
Makes sense, fixed it!